src/service/BookingService.java

```diff
@@ -30,6 +31,22 @@ public BookingService(BookingDataDAO bookingDataDAO,GuestDataDAO guestDataDAO) {
 	}
 
 	public Integer saveBooking(LocalDateTime entryDate, LocalDateTime departureDate, PaymentMethodDTO methodPayment)  {
+		if (entryDate == null) {
+			throw new IllegalArgumentException("entryDate was null");
+		}
+		if (departureDate == null) {
+			throw new IllegalArgumentException("departure was null");
+		}
+		if (methodPayment == null) {
+			throw new IllegalArgumentException("method payment was null");
+		}
+		if (entryDate.isBefore(LocalDateTime.now())) {
+			throw new IllegalArgumentException("entry date is incorrect");
```

Comment:
Please use a more accurate message such as `"Entry date can't be before today."`.

Additionally please create a test scenario in which the entry date is today's date but minus one hour, let me know what happens.